### PR TITLE
Create `eslint-config-mic`

### PR DESCRIPTION
**Pull request by danmakenoise**
This PR creates `eslint-config-mic`, which can be used independently of the `miclint` CLI. This will allow us to create better individualized solutions for repos, and have easier integration into editors for both our development team, and open-source teams.

Check out the new [readme](https://github.com/micnews/miclint/tree/create-configs/eslint-config-mic) in the `eslint-config-mic` folder for directions to use!

### CODE REVIEW

File: options.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   tagline: 'Mic lint',
   eslint: eslint,
   eslintConfig: {
-    configFile: path.join(__dirname, 'eslintrc.json')
+    configFile: path.join(__dirname, 'eslint-config-mic/index.js')
```
Comment (ellell) [inline options.js:13]: @danmakenoise just want to make sure this is working even though it's not a .json file?

**Review by ellell — APPROVED** [review]
@danmakenoise looks good, just added one question! 

Also, should we add a reference to the sub package readme in the main readme here, just to make it clear that exists in here?

File: options.js (hunk 1 above)
Comment (danmakenoise) [reply options.js:13]: @ellell I tested locally and it seemed just fine! I'll do a triple check after publishing.

**Comment by danmakenoise** [comment]
@ellell will do on the readme part, then I'll merge!

**Pull request merged**